protocols/util/factories.py

```diff
@@ -0,0 +1,47 @@
+import factory.fuzzy
+import itertools
+from protocols.ga4gh_3_0_0 import Variant, Call
+from .avro_factory import FactoryAvro
```

Comment:
Can we please have builtins then third party then internal imports ? So something like:
```
import itertools

import factory.fuzzy

from .avro_factory import FactoryAvro
from protocols.ga4gh_3_0_0 import Call
from protocols.ga4gh_3_0_0 import Variant
```